internal/npo-ext.owl

```diff
@@ -3185,7 +3197,50 @@
         <rdfs:subClassOf rdf:resource="http://purl.bioontology.org/ontology/npo#NPO_1999"/>
         <rdfs:label xml:lang="en">disc centrifuge</rdfs:label>
     </owl:Class>
+    
+    <!-- http://purl.enanomapper.org/onto/ENM_0000104 -->
+        
+    <owl:Class rdf:about="http://purl.enanomapper.org/onto/ENM_0000104">
+        <rdfs:subClassOf rdf:resource="http://purl.bioontology.org/ontology/npo#NPO_1430"/>
+        <rdfs:label xml:lang="en">Full field transmission X-ray microscopy</rdfs:label>
+    </owl:Class>
+    
+    <!-- http://purl.enanomapper.org/onto/ENM_0000105 -->
+        
+    <owl:Class rdf:about="http://purl.enanomapper.org/onto/ENM_0000105">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/PATO_0000033"/>
+        <rdfs:label xml:lang="en">Chloride concentration</rdfs:label>
+    </owl:Class>
+    
+    <!-- http://purl.enanomapper.org/onto/ENM_0000106 -->
```

Comment:
Tricky one, do not know if Solid matrix should be added as a subclass of solid state (NPO 1511)

Reply:
I would rather add it under "composite material" https://en.wikipedia.org/wiki/Composite_material